packages/api/src/routes/v2/story.ts

```diff
@@ -24,7 +24,7 @@ export default (app: Router): void => {
      *         schema:
      *           type: string
      *         required: true
-     *         description: media mimetype
+     *         description: media mimetype, separated by semicolon (png;jpg).
```

Comment:
I think it costs the same for frontend (maybe easier) and is easier for backend, if you accept something like `?mime=png&mime=jpg&mime=jpg` because it is transformed in an array when received by the server. Same as we did on story list on v2 where you could filter by country using `?country=PT&country=BR` or by many communities like `?community=2&community=5`. Keeps code cleaner and consistent 💪

Reply:
much better in this way!
fixed.